src/mmgs/inout_s.c

```diff
@@ -171,7 +172,7 @@ int MMGS_loadMesh(MMG5_pMesh mesh, const char *filename) {
     }
     MMG_FREAD(&mesh->ver,MMG5_SW,1,inm);
     if(iswp) mesh->ver = MMG5_swapbin(mesh->ver);
-    while(fread(&binch,MMG5_SW,1,inm)!=0 && binch!=54 ) {
+    while(fread(&binch,4,1,inm)!=0 && binch!=54 ) {
```

Comment:
Why removing the macro here? Is it not the same?

Reply:
This is a mistake. I reverted this change.